cli/start.go

```diff
@@ -189,6 +189,9 @@ func runStart(_ *cobra.Command, _ []string) error {
 	}
 
 	info := build.GetInfo()
+	// We log build information to stdout (for the short summary), but also
+	// to stderr to coincide with the full logs.
+	log.Infof("[build] %s @ %s (%s)", info.Tag, info.Time, info.GoVersion)
```

Comment:
:thumbs_up:

On Thu, Apr 21, 2016 at 11:30 AM marc <notifications@github.com> wrote:

> In cli/start.go
> <https://github.com/cockroachdb/cockroach/pull/6195#discussion_r60602031>:
>
> > @@ -189,6 +189,9 @@ func runStart(_ *cobra.Command, _ []string) error {
> >  	}
> >
> >  	info := build.GetInfo()
> > +	// We log build information to stdout (for the short summary), but also
> > +	// to stderr to coincide with the full logs.
> > +	log.Infof("[build] %s @ %s (%s)", info.Tag, info.Time, info.GoVersion)
>
> sure thing, done. I've also moved it back to the beginning. After trying a
> few different place, I found that there's something satisfying about having
> the first log line be the version string.
>
> —
> You are receiving this because you were assigned.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/6195/files/95a5e24b3162b5c91312d0ff840c4ec3eb66c4bf#r60602031>
>
-- 

-- Tobias